pkg/login/other_logins_check.go

```diff
@@ -0,0 +1,151 @@
+package login
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	logger "github.com/sirupsen/logrus"
+
+	authenticationapi "k8s.io/api/authentication/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
```

Comment:
Not against hardcoding the team prefix, but this may cause onboarding new teams and supporting the same CLI for fed ramp backplane API's 
If we can fetch the list via the bp-API endpoint, that would be easy to maintain ( Just my thoughts )

Reply:
Is this available as an endpoint on the backplane API already or would that be net new? Happy to use that if it already exists. 

I'll definitely be checking here to make sure this works with FedRAMP clusters here too.

Reply:
There is no current config API yet , but we are planning to add config API (https://issues.redhat.com/browse/OSD-25654 ) 
We can keep it on the bp-cli side for now

Reply:
Makes sense! Once that API endpoint is done then having this be requested dynamically makes total sense!